lib/index.js

```diff
@@ -1,39 +1,49 @@
+'use strict';
+
 var AWS = require('aws-sdk');
-AWS.config.region = process.env.AWS_REGION;
 var ses = new AWS.SES();
-var path               = require('path');       // resolve template files
-var fs                 = require('fs');         // open tempalte files
-var Handlebars         = require('handlebars'); // compile templates
-var COMPILED_TEMPLATES = {};                    // cache compiled templates
+var path = require('path');             // -> resolve template files
+var fs = require('fs');                 // -> open template files
+var Handlebars = require('handlebars'); // -> compile templates
+var COMPILED_TEMPLATES = {};            // -> cache compiled templates
+
+AWS.config.region = process.env.AWS_REGION;
```

Comment:
can this be configured down here _after_ `SES` has been instantiated?
(_good ol OOP, hey...?_) 😆